tests/integration/records/serializers/json/test_authors.py

```diff
@@ -0,0 +1,332 @@
+    record_control_number = record.json["control_number"]
+
+    expected_status_code = 200
+    expected_id = record_control_number
+    expected_metadata = {
+        "advisors": [
+            {
+                "degree_type": "other",
+                "ids": [{"schema": "INSPIRE ID", "value": "INSPIRE-00070625"}],
+                "name": "Callan, Curtis G.",
+            }
+        ],
+        "arxiv_categories": ["hep-th", "gr-qc"],
+        "control_number": 999_108,
+        "deleted": False,
+        "email_addresses": [{"current": True, "value": "malda@ias.edu"}],
+        "facet_author_name": "J.M.Maldacena.1_Juan Martin Maldacena",
+        "ids": [
+            {"schema": "INSPIRE ID", "value": "INSPIRE-00304313"},
+            {"schema": "INSPIRE BAI", "value": "J.M.Maldacena.1"},
+            {"schema": "ORCID", "value": "0000-0002-9127-1687"},
+            {"schema": "SPIRES", "value": "HEPNAMES-193534"},
+        ],
+        "name": {
+            "name_variants": ["Maldacena, Juan Martin"],
+            "preferred_name": "Juan Martin Maldacena",
+            "value": "Maldacena, Juan Martin",
+        },
+        "positions": [
+            {
+                "current": True,
+                "display_date": "2001-present",
+                "institution": "Princeton, Inst. Advanced Study",
+                "rank": "SENIOR",
+            },
+            {
+                "display_date": "1997-2001",
+                "institution": "Harvard U.",
+                "rank": "SENIOR",
+            },
+            {
+                "display_date": "1996-1997",
+                "institution": "Rutgers U., Piscataway",
+                "rank": "POSTDOC",
+            },
+            {"display_date": "1992-1996", "institution": "Princeton U.", "rank": "PHD"},
+            {
+                "display_date": "1988-1991",
+                "institution": "Cuyo U.",
+                "rank": "UNDERGRADUATE",
+            },
+        ],
+        "legacy_creation_date": "1999-05-04",
+        "legacy_version": "20160711200442.0",
+        "public_notes": [
+            {"value": "Fundamental Physics Price 2012"},
+            {"value": "Dirac Medal 2008"},
+            {"value": "Heineman Prize 2007"},
+        ],
+        "should_display_positions": True,
+        "status": "active",
+        "stub": False,
+        "urls": [{"value": "http://www.sns.ias.edu/~malda"}],
+    }
+    response = api_client.get(f"/authors/{record_control_number}", headers=headers)
+
+    response_status_code = response.status_code
+    response_data = json.loads(response.data)
+
+    assert expected_status_code == response_status_code
+    assert expected_metadata == response_data["metadata"]
+    assert expected_id == response_data["id"]
+    assert response_data["created"] is not None
+    assert response_data["updated"] is not None
+
+
+def test_authors_json_without_login(api_client, db, create_record_factory):
```

Comment:
do we care about ``json``? shouldn't be ``detail`` with `` headers = {"Accept": "application/vnd+inspire.record.ui+json"}``?

Reply:
we care, because raw json apis also shouldn't return hidden emails.